homeassistant/components/ping/binary_sensor.py

```diff
@@ -212,7 +212,7 @@ def __init__(self, hass, host, count, privileged) -> None:
             "-q",
             "-c",
             str(self._count),
-            "-W1",
+            f"-W{ICMP_TIMEOUT}",
```

Comment:
Do we need to adjust the polling interval?

Reply:
it's not the polling interval
```bash
$ ping -h 
[...]
-W <timeout>       time to wait for response
```

Reply:
ah, now i get it ... the default polling interval for device tracker is 12s so we do not need to increase it.
Only in case the user set an own interval in YAML (_key `interval_seconds`_), but this is not a documented (_but still an available_) option

Reply:
<https://www.home-assistant.io/integrations/device_tracker> It's definitely documented

Reply:
but not in the [ping](https://www.home-assistant.io/integrations/ping#configuration) docs 😕

Reply:
anyway, in this case I would add a breaking change message, to inform the users, that the increased timeout may need an adjustment of their own scan interval settings